python/ray/train/lightning/lightning_trainer.py

```diff
@@ -308,6 +309,17 @@ def __init__(
         preprocessor: Optional[Preprocessor] = None,
         resume_from_checkpoint: Optional[Checkpoint] = None,
     ):
+        run_config = run_config or RunConfig()
+        lightning_config = lightning_config or LightningConfigBuilder().build()
+
+        self._check_checkpoint_configs(
+            ptl_ckpt_config=lightning_config["_model_checkpoint_config"],
+            air_ckpt_config=run_config.checkpoint_config,
+        )
+
+        # Disable strict checking to allow metric reporting at different frequencies
+        os.environ["TUNE_DISABLE_STRICT_METRIC_CHECKING"] = "1"
```

Comment:
😓 
any chance we can disable the csv writer so we don't have to do this hack here?